serverless/openvino/omz/intel/semantic-segmentation-adas-0001/nuclio/model_handler.py

```diff
@@ -8,6 +9,14 @@
 from skimage.measure import approximate_polygon, find_contours
 from model_loader import ModelLoader
 
+
+def to_cvat_mask(box: list, mask):
+    xtl, ytl, xbr, ybr = box
+    flattened = mask[ytl:ybr + 1, xtl:xbr + 1].flat[:].tolist()
+    flattened.extend([xtl, ytl, xbr, ybr])
+    return flattened
```

Comment:
Would it be an option, to put shared code to ``serverless/common/shared.py`` or something like this?

Reply:
Moved `to_cvat_mask` to a separate file